source/technical/images.rst

```diff
@@ -10,6 +10,40 @@ To work around this limitation, we provide the ``cc-snapshot`` utility that you
 
 The image service on Chameleon uses `OpenStack Glance <https://docs.openstack.org/glance/latest/>`_. This documentation demonstrates how to accomplish common tasks with *Images* using the GUI and the CLI.
 
+__________________________________
+Chameleon Supported Images
+__________________________________
+
+There are a number of images built and supported by the Chameleon team,
+specifically:
+- CC-Ubuntu24.04
+- CC-Ubuntu24.04-CUDA
+- CC-Ubuntu24.04-ROCm
+- CC-Ubuntu24.04-ARM64
+- CC-Ubuntu22.04
+- CC-Ubuntu22.04-CUDA
+- CC-Ubuntu22.04-ARM64
+- CC-Centos9-Stream
+
+The CUDA images contain various settings, software, and drivers specifically
+for NVIDIA GPU nodes. The ROCm images contain similar settings, software,
+and drivers for AMD GPU nodes. And finally, the ARM64 images are images
+specifically built with ARM support for ARM nodes. Non-ARM images all
+assume x86-based architectures.
+
```

Comment:
Maybe link to the appliance catalog for the additional details?

Reply:
done